mmv1/products/gkehub2/Fleet.yaml

```diff
@@ -95,3 +95,49 @@ properties:
           - READY
           - DELETING
           - UPDATING
+  - !ruby/object:Api::Type::NestedObject
+    name: "defaultClusterConfig"
+    description: The default cluster configurations to apply across the fleet.
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: "securityPostureConfig"
+        description: Enable/Disable Security Posture features for the cluster.
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: "mode"
+            description: Sets which mode to use for Security Posture features.
+            values:
+              - MODE_UNSPECIFIED
```

Comment:
Generally we leave off the UNSPECIFIED values for enums in Terraform since they are not meaningful for users to set.

Reply:
Done.